engine/src/test/java/org/camunda/bpm/engine/test/bpmn/event/timer/StartTimerEventTest.java

```diff
@@ -1661,6 +1781,14 @@ protected void executeAllJobs() {
 
   }
 
+  protected void moveByHours(int hours) throws Exception {
+    ClockUtil.setCurrentTime(new Date(ClockUtil.getCurrentTime().getTime() + ((hours * 60 * 1000 * 60) + 5000)));
+    JobExecutor jobExecutor = processEngineConfiguration.getJobExecutor();
+    jobExecutor.start();
+    Thread.sleep(1000);
+    jobExecutor.shutdown();
+  }
```

Comment:
🔧 How about we rewrite this a bit using the `testRule` to simply execute existing jobs instead of starting the job executor and letting the main thread sleep?

Reply:
`#executeAvailableJobs(false)` disables the recursion, executes only the currently available job, so it works for this test class. Will adjust the helper method.